package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ng2-mask",
+  "name": "ng2-mask-unofficial",
   "version": "1.0.5",
-  "description": "awesome ng2 mask",
+  "description": "awesome ng2 mask - unofficial",
```

Comment:
We can modify. It's just because i needed to create a new package in npmjs, to get more dynamic updated ng2-mask library to the project i'm in at my work. But it's just a reference to the 'official' (this) project, with some functionality in test stage.